python-checks/src/main/java/org/sonar/python/checks/CommentRegularExpressionCheck.java

```diff
@@ -19,24 +19,23 @@
  */
 package org.sonar.python.checks;
 
-import com.sonar.sslr.api.Token;
-import com.sonar.sslr.api.Trivia;
 import java.util.regex.Pattern;
 import org.sonar.check.Rule;
 import org.sonar.check.RuleProperty;
-import org.sonar.python.PythonCheckAstNode;
+import org.sonar.python.PythonSubscriptionCheck;
+import org.sonar.python.api.tree.Token;
+import org.sonar.python.api.tree.Tree;
+import org.sonar.python.api.tree.Trivia;
+
+@Rule(key = "CommentRegularExpression")
+public class CommentRegularExpressionCheck extends PythonSubscriptionCheck {
 
-@Rule(key = CommentRegularExpressionCheck.CHECK_KEY)
-public class CommentRegularExpressionCheck extends PythonCheckAstNode {
-  public static final String CHECK_KEY = "CommentRegularExpression";
   private static final String DEFAULT_REGULAR_EXPRESSION = "";
   private static final String DEFAULT_MESSAGE = "The regular expression matches this comment";
 
   private Pattern pattern = null;
 
-  @RuleProperty(
-    key = "regularExpression",
-    defaultValue = "" + DEFAULT_REGULAR_EXPRESSION)
+  @RuleProperty(key = "regularExpression")
```

Comment:
Why did you remove the `defaultValue`?

Reply:
It was redundant, as DEFAULT_REGULAR_EXPRESSION is already set to ""